### PR TITLE
Adding testing suite for unit testing.

### PR DESCRIPTION
**Pull request by juliussneezer04**
Will be easier and scalable to use a testing suite for future increments as well.
MochaJS is an easy-to-use framework that can be utilized for this.
Pull request contains code from @vigonometry 's #17 Pull Request.

### CODE REVIEW

File: helpers/test/chunkify.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        assert.deepEqual(actualChunkifiedMessage, ["Valid Message"]);
+        done();
+    });
+
+    /**
+     * Tests whether large valid messages containing CHAR_LIMIT characters is un-modified.
+     */
+    it('Does not modify valid large messages', function(done) {
+        const testFilePath = "helpers/test/out_newline.txt";
+        const VALID_LARGE_TEST_MESSAGE = readTextFile(testFilePath);
```
Comment (nicktohzyu) [inline helpers/test/chunkify.test.js:28]: check that this contains correct num of chars

File: helpers/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+=============
+Function was added to split text (as strings) into smaller chunks of no more than characters specified by the end user. Function was subsequently exported. (_Recursive Implementation_)
+
+test
+=========
+Directory with relevant testing environment (chunkify_test.js) and python script to generate user inputs for testing (gen_chunkify_test_inputs.py)
+
+
+
+## Testing with MochaJS
```
Comment (nicktohzyu) [inline helpers/README.md]: move this to main readme?

File: helpers/test/chunkify.test.js (hunk 1 above)
Comment (juliussneezer04) [reply helpers/test/chunkify.test.js:28]: amended to 4096 👍

**Comment by juliussneezer04** [comment]
Closing branch to re-open from master.

**Pull request closed without merging**